src/test/java/guitests/InvalidLoginTest.java

```diff
@@ -32,9 +31,8 @@ public void invalidLoginTest() throws InterruptedException {
 
     @Test
     public void emptyLoginTest() throws InterruptedException {
-        TextField repoOwnerField = find("#repoOwnerField");
-        doubleClick(repoOwnerField);
-        doubleClick(repoOwnerField);
+        find("#repoOwnerField");
+        selectAll();
```

Comment:
I can make such function but I don't find so many of them from replacing
all with selectAll. In fact, I don't find any thay does it with waiting.
On Nov 8, 2015 7:00 PM, "Darius Foo" <notifications@github.com> wrote:

> In src/test/java/guitests/InvalidLoginTest.java
> <https://github.com/HubTurbo/HubTurbo/pull/1123#discussion_r44223506>:
>
> > @@ -32,9 +31,8 @@ public void invalidLoginTest() throws InterruptedException {
> >
> >      @Test
> >      public void emptyLoginTest() throws InterruptedException {
> > -        TextField repoOwnerField = find("#repoOwnerField");
> > -        doubleClick(repoOwnerField);
> > -        doubleClick(repoOwnerField);
> > +        find("#repoOwnerField");
> > +        selectAll();
>
> An abstraction that could be nice is to wait until a node appears, find,
> click, select all, then type a string. Seems like this is frequently done
> but I don't know exactly how much.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/HubTurbo/HubTurbo/pull/1123/files#r44223506>.
>